content/chapters/template-chapter/lab/README.md

```diff
@@ -62,1302 +33,25 @@ student@os:~$ file /usr/bin/ls
-A process is a logical group of such tasks.
-
-We can sum up what we've learned so far by saying that processes are better used for separate, independent work, such as the different connections handled by a server.
-Conversely, threads are better suited for replicated work: when the same task has to be performed on multiple cores.
-However, replicated work can also be suited for processes.
-Distributed applications, however, leverage different processes as this allows them to run on multiple physical machines at once.
-This is required by the very large workloads such applications are commonly required to process.
-
-These rules are not set in stone, though.
-Like we saw in the `apache2` example, the server uses multiple threads as well as multiple processes.
-This provides a degree of stability - if one worker thread crashes, it will only crash the other threads belonging to the same process - while still taking advantage of the light resource usage inherent to threads.
-
-This kind of trade-offs are a normal part in the development of real-world applications.
-
-## Copy-on-Write
-
-So far you know that the parent and child process have separate virtual address spaces.
-But how are they created, namely how are they "separated"?
-And what about the **physical address space**?
-Of course we would like the stack of the parent, for example, to be physically distinct from that of the child so they can execute different functions and use different local variables.
-
-But should **all** the PAS of the parent be distinct from that of the child?
-What about some read-only memory sections, such as `.text` and `.rodata`?
-And what about the heap, where the child _may_ use some data previously written by the parent and then override it with its own data.
-
-The answer to all of these questions is a core mechanism of multi-process operating systems called **Copy-on-Write**.
-It works according to one very simple principle:
-> The VAS of the child process initially points to the same PAS as that of the parent.
-> A (physical) frame is only duplicated by the child when it attempts to **write** data to it.
-
-This ensures that read-only sections remain shared, while writable sections are shared as long as their contents remain unchanged.
-When changes happen, the process making the change receives a unique frame as a modified copy of the original frame _on demand_.
-
-In the image below we have the state of the child and parent processes right after `fork()` returns in both of them.
-See how each has its own VAS, both of them being mapped to (mostly) the same PAS.
-
-
-When one process writes data to a writeable page (in our case, the child writes to a heap page), the frame to which it corresponds is first duplicated.
-Then the process' page table points the page to the newly copied frame, as you can see in the image below.
+1. You can number the exercises as such
+And have the text roll over to the next line.
 
+2. This is a second numbered exercise
```

Comment:
```suggestion
1. This is a second numbered exercise
```
Only use `1.` for numbered lists. It's easier to add new list items between existing ones. You can actually state this here in `README.md` as a strong recommendation.

Reply:
I will add this to my TODO list.